runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3030,6 +3035,34 @@ gcInitializeDefaults(J9JavaVM* vm)
 	return J9VMDLLMAIN_FAILED;
 }
 
+void
+initializeIndexableObjectHeaderSizes(J9JavaVM* vm) {
```

Comment:
I'm on a fence if this belongs to VM or GC code. @dmitripivkine, opinion?

Reply:
This code uses `omrVM->gcPolicy` so, it should be called after `configurateGCWithPolicyAndOptions()` (which is called in the same function `gcInitializeDefaults()` earlier). I am not sure when general initialization of `J9JavaVM` is done. I think despite only VM fields are initialized here it is safer to keep it here to control order of initialization.